### ISSUE TITLE
[Chore] Update Issue Code Templates

### ISSUE THREAD

**Issue by DintaMel** [Esri/calcite-components#13879]
**monday.com sync:** #11207737661

### Which Component

Unknown / Not Sure

### Description

Due to CDN changes, the codepen templates in the Bug and Accessibility templates should be updated.

Refer to issue 1407 in the docs repo.

### Resources

CDN changes occured with issue #13266. Now that Calcite v5.0 is released, the codepen should be updated.

**Comment by geospatialem** [comment]
Verified the bug and accessibility bug issue templates now have the updated Codepen template using the cdn reference of:

```html
<script type="module" src="https://cdn.jsdelivr.net/npm/@esri/calcite-components@latest/dist/cdn/index.js"></script>
```
